src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -312,6 +337,25 @@ class WorkspaceMembersPage extends React.Component {
                         />
                     </View>
                 </View>
+                <FixedFooter
+                    style={[styles.pt3, styles.pb3]}
+                >
+                    this.props.network.isOffline &&
```

Comment:
Needs to be: 
```Javascript
{this.props.network.isOffline && (
   <View> 
   ...
   </View> 
)} 
```
 otherwise, you get this: 

![image](https://user-images.githubusercontent.com/16747822/153936094-b0063799-ec3f-411a-bb55-2ae280639ce1.png)

Reply:
Okay